common/views/components/IIIFSearchWithin/IIIFSearchWithin.tsx

```diff
@@ -122,21 +120,32 @@ const IIIFSearchWithin: FunctionComponent<Props> = ({
           getSearchResults();
         }}
       >
-        {' '}
+        <span
+          className={classNames({
+            [font('hnm', 5)]: true,
+          })}
+        >
+          Search within this item
+        </span>
         <SearchInputWrapper className="relative">
           <TextInput
             id={'searchWithin'}
-            label={'Search within this item'}
-            placeholder={'Search within this item'}
+            label={'Enter keyword'}
             name="query"
             value={value}
             setValue={setValue}
             required={true}
+            aria-label={`Search within this item`}
```

Comment:
We're were trying to [store our labels in a separate file to make it easier to audit](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/text/arial-labels.ts), not sure if this is the right idea. What do you think?

Reply:
that seems like a very sensible idea